src/overlays/actors/ovl_Bg_Po_Syokudai/z_bg_po_syokudai.c

```diff
@@ -21,11 +46,128 @@ const ActorInit Bg_Po_Syokudai_InitVars = {
     (ActorFunc)BgPoSyokudai_Update,
     (ActorFunc)BgPoSyokudai_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Po_Syokudai/BgPoSyokudai_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Po_Syokudai/BgPoSyokudai_Destroy.s")
+static InitChainEntry sInitChain[] = {
+    ICHAIN_VEC3F_DIV1000(scale, 1000, ICHAIN_STOP),
+};
+
+extern Gfx D_060003A0[]; // Display List
+extern Gfx D_0404D4E0[]; // Display List
+
+void BgPoSyokudai_Init(Actor* thisx, GlobalContext* globalCtx) {
+    BgPoSyokudai* this = THIS;
+
+    void* pad;
+
+    Actor_ProcessInitChain(&this->actor, sInitChain);
+
+    this->unk_14C = (THIS->actor.params >> 0x8) & 0xFF;
+    this->actor.params &= 0x3F;
+
+    this->actor.colChkInfo.mass = 0xFF;
+
+    this->lightNode = LightContext_InsertLight(globalCtx, &globalCtx->lightCtx, &this->lightInfo);
+    Lights_PointGlowSetInfo(&this->lightInfo, (s16)this->actor.posRot.pos.x, ((s16)this->actor.posRot.pos.y) + 0x41,
```

Comment:
`0x41` is coordinates-related so base 10 (decimal) would make more sense for it
65 also is a "nicer looking" value